client/blocks/support-article-dialog/style.scss

```diff
@@ -4,26 +4,28 @@
 }
 
 .support-article-dialog.dialog.card {
+	margin: 0 auto;
 	position: absolute;
 	top: 0;
 	bottom: 0;
-	left: 0;
-	right: 0;
+	width: 100%;
+	height: 100%;
 	max-width: 100%;
+	max-height: 100%;
 
 	@include breakpoint-deprecated( '>660px' ) {
 		top: 5%;
 		bottom: 5%;
-		left: 5%;
-		right: 5%;
 		width: 90%;
+		height: 90%;
+	}
+
+	@include breakpoint-deprecated( '>860px' ) {
```

Comment:
I haven't investigated it, but I'm seeing this error in the console when running calypso locally.
```
WARNING: ERROR in breakpoint-deprecated( >860px ): You can only use these sizes[  480px 660px 800px 960px 1040px 1280px 1400px ] using the following syntax [ <480px >480px 480px-660px ]
    client/assets/stylesheets/shared/mixins/_breakpoints.scss 50:5  breakpoint-deprecated()
    client/blocks/support-article-dialog/style.scss 24:2            root stylesheet
```

Reply:
Yup this seems to break a custom style lint @michaeldcain maybe we can suppress the warning if this change is required?
any tips on how to do this @tyxla @jsnajdr?  curious to know how we handle this type of custom requirement if it ever comes up 🙂

Reply:
Can you use just the `>960px` breakpoint? There is very little difference.

Right now, the style contains two identical breakpoint rules, just next to each other, one for `>860px` and the other one for `>960px`.

Mysteriously enough, the compiled stylesheet (in dev mode) contains only the `>960px` one. The `>860px` one disappeared from the output, not sure why.

Reply:
> Mysteriously enough, the compiled stylesheet (in dev mode) contains only the `>960px` one. The `>860px` one disappeared from the output, not sure why.

Looks like the scss helper does not create any new output if the breakpoint is not "approved": [Link](https://github.com/Automattic/wp-calypso/blob/09a077b5c1695cc146caccfae82a49b025b370ce/client/assets/stylesheets/shared/mixins/_breakpoints.scss#L44). I remember running into this once before

Reply:
Huh - I didn't see this error when developing. Sorry. I'll add it to the 660px breakpoint above

Reply:
Fix in https://github.com/Automattic/wp-calypso/pull/54619